lib/plugin-registry.coffee

```diff
@@ -35,22 +36,25 @@ module.exports =
     @plugins[code] = plugin
 
     if @enabled
-      @addConfigForPlugin code, plugin, key
+      @addConfigForPlugin(code, plugin, key)
       @observePlugin code, plugin, key
 
   addConfigForPlugin: (code, plugin, key) ->
+    return if atom.config.get(key) == undefined
```

Comment:
This is wrong, as if it stops here the plugin config won't be registered. So the user won't be able to enable to plugin.

Reply:
Yes it's true I didn't realize.